src/components/Table/index.tsx

```diff
@@ -168,7 +166,9 @@ function Table<D, K extends string | number, C extends Column<D, K, any, any>>(
         sum(columns.map((c) => columnWidths[c.id]))
     ), [columnWidths, columns]);
 
-    const isEmpty = isNotDefined(data) || data.length === 0;
+    const isEmpty = isNotDefined(data)
+        || data.length === 0
+        || Object.keys(columnWidths).length === 0;
```

Comment:
Let's add NOTE why we are not showing information if we do not have columnWidths

Reply:
I've no idea, it was already there. I just moved the lines!